hadoop-ozone/dist/src/main/smoketest/admincli/scmha.robot

```diff
@@ -25,4 +25,4 @@ Test Timeout        5 minutes
 *** Test Cases ***
 Run scm roles
     ${output} =         Execute          ozone admin scm roles
-                        Should contain   ${output}  [scm:9865]
+                        Should Match Regexp   ${output}  [scm:9865:(LEADER|FOLLOWER)]
```

Comment:
Is it possible to verify that we have 1 LEADER & 2 FOLLOWER strings in the response?

Reply:
Thanks @avijayanhwx for the review and great comment. As discussed, this test was actually executed from ozonesecure with a 1 node SCM cluster. Therefore we will not have 1 LEADER and 2 FOLLOWER.

Also, had some discussion with @GlenGeng, there is a need to differentiate a single node SCM cluster from an SCM HA cluster with only 1 existing SCM node. In the later case, the command will print out something like this: [scm1:9865:LEADER] and therefore we'll know that this is a HA setup. Therefore the implementation of "getRatisRoles" remains unchanged from the previous version for now. Thanks again!